content/articles/2021-06-06-poliastro-google-summer-of-code-2021.md

```diff
@@ -0,0 +1,24 @@
+Title: GSoC 2021, the journey begins!
+Date: 2021-06-06 11:00 IST
+Category: GSOC
+Tags: GSOC, GSOC21, poliastro
+slug: 2021-06-06-poliastro-google-summer-of-code-2021
+lang: en
+Author: Yash
+
+
+# The initial days…
+
+Hi there! I am Yash, and I will be spending this summer with Poliastro as part of GSoC 2021 to add some event detection capabilities. It was around December when I first learned about Poliastro, and it gradually got me fascinated by orbital mechanics.
```

Comment:
I know it's silly, but poliastro is not capitalized 😄

Reply:
Oh! I forgot to lowercase it earlier..Thanks